app/client/keybase/debug/keystore.go

```diff
@@ -16,8 +16,9 @@ import (
 
 const (
 	// NOTE: This is the number of validators in the private-keys.yaml manifest file
-	numValidators      = 999
-	debugKeybaseSuffix = "/.pocket/keys"
+	numValidators                      = 999
+	debugKeybaseSuffix                 = "/.pocket/keys"
+	debugKeybaseImportConcurrencyLimit = 4
```

Comment:
4 takes 350-400MiB on my machine which sounds acceptable.

Reply:
You should make this part of the comment!